elyra/pipeline/processor_kfp.py

```diff
@@ -276,11 +307,14 @@ def _cc_pipeline(self, pipeline, pipeline_name):
 
             # create pipeline operation
             notebook_ops[operation.id] = NotebookOp(name=operation.name,
+                                                    pipeline_name=pipeline.name,
```

Comment:
Shouldn't the variable `pipeline_name` be used here and not `pipeline.name`?  This seems to support my previous question.

Reply:
Yep. Fixed